torch/fx/experimental/proxy_tensor.py

```diff
@@ -405,7 +423,7 @@ def wrap_with_proxy(e: object, proxy: _NestedProxys, constant: Optional[_NestedT
             assert isinstance(proxy, Proxy)
             # NB: eagerly set meta here, so that the numbering is in order
             set_meta(proxy, e)
-            set_proxy_slot(e, tracer, lambda: proxy)
+            set_proxy_slot(e, tracer, thunkify(lambda: proxy))
```

Comment:
Does this really make a difference? You just change the lambda capturing proxy with your class holding onto it in .r ?

Reply:
The interface for Thunk is different, it's not callable you call .force() on it